client/src/components/comparison/LeftArrow.jsx

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+const LeftArrow = (props) => {
+  return (
+    <div className='backArrow' onClick={this.props.goToPrevSlide}>
```

Comment:
You don't need `this` here. Within a functional component, you can just refer to `props`. In a class component, you would have to use `this.props`.

Reply:
oops, removed the 
'this.'